easystacks/software.eessi.io/2023.06/eessi-2023.06-eb-4.9.2-2023b.yml

```diff
@@ -15,3 +15,6 @@ easyconfigs:
       options:
         # see https://github.com/easybuilders/easybuild-easyconfigs/pull/21034
         from-commit: 76e7fc6657bab64bfbec826540a3a8f0040258f2
+  - xxd-9.1.0307-GCCcore-13.2.0.eb
+  - HTSlib-1.19.1-GCC-13.2.0.eb
```

Comment:
While it doesn't hurt to list these, there's no need to, since they're (build) dependencies of `STAR-2.7.11b-GCC-13.2.0.eb`, so let's remove these lines?

```suggestion
```

Reply:
Yes, sure, feel free to revert this. I'm still figuring out the whole process ...